opencti-platform/opencti-graphql/tests/02-integration/02-resolvers/malwareAnalysis-test.js

```diff
@@ -0,0 +1,209 @@
+import { describe, expect, it } from 'vitest';
+import gql from 'graphql-tag';
+import { queryAsAdmin } from '../../utils/testQuery';
+
+const LIST_QUERY = gql`
+  query malwareAnalyses(
+    $first: Int
+    $after: ID
+    $orderBy: MalwareAnalysesOrdering
+    $orderMode: OrderingMode
+    $filters: [MalwareAnalysesFiltering!]
+    $filterMode: FilterMode
+    $search: String
+  ) {
+    malwareAnalyses(
+      first: $first
+      after: $after
+      orderBy: $orderBy
+      orderMode: $orderMode
+      filters: $filters
+      filterMode: $filterMode
+      search: $search
+    ) {
+      edges {
+        node {
+          id
+          result_name
+          product
+        }
+      }
+    }
+  }
+`;
+
+const READ_QUERY = gql`
+  query malwareAnalysis($id: String!) {
+    malwareAnalysis(id: $id) {
+      id
+      standard_id
+      result_name
+      product
+      toStix
+    }
+  }
+`;
+
+describe('MalwareAnalysis resolver standard behavior', () => {
```

Comment:
Have you deleted everything you have created at the end of your test ?

Reply:
I think so.
The malware analysis and its relation are deleted in the last two tests